theorems/T000400.md

```diff
@@ -2,13 +2,12 @@
 uid: T000400
 if:
   and:
-    - P000147: true
+    - P000085: true
     - P000036: true
 then:
   P000060: true
-refs:
-- doi: 10.1016/0016-660x(72)90026-8
-  name: A topological view of P-spaces (A. Misra)
 ---
 
-See proposition 5.1 in {{doi:10.1016/0016-660x(72)90026-8}}.
+The case of the empty space is obvious.
+Assume $X$ is a nonempty, connected and basically disconnected space. Fix continuous function $f:X\to \mathbb R$. For arbitrary $x\in X$ and $\varepsilon>0$ define $g_{x,\varepsilon}(y)=f(y)-f(x)+\varepsilon$.
+Then the set $A =\overline{\{y\in X: g_{x,\varepsilon}(y)>0\}}$ is nonemplty clopen and henceforth equals $X$. Since $X=A\subset g_{x,\varepsilon}^{-1}([0,+\infty))$, we have $f(y)\geq f(x)-\varepsilon$ for any $x,y\in X$ and $\varepsilon>0$, implying that $f$ is constant.
```

Comment:
```suggestion
Assume $X$ is a nonempty, connected and basically disconnected space. Let $f:X\to \mathbb R$ be a continuous function. For arbitrary $x\in X$ and $\varepsilon>0$ define $g_{x,\varepsilon}(y)=f(y)-f(x)+\varepsilon$.
Then the set $A =\overline{\{y\in X: g_{x,\varepsilon}(y)>0\}}$ is nonempty clopen and hence equals $X$. Since $X=A\subset g_{x,\varepsilon}^{-1}([0,+\infty))$, we have $f(y)\geq f(x)-\varepsilon$ for any $x,y\in X$ and $\varepsilon>0$, implying that $f$ is constant.
```